### DISCUSSION TITLE
Chia transaction fee.

### DISCUSSION THREAD

**Discussion by lacrymatoire** [Chia-Network/chia-blockchain#5974] [Support]
Hi, a newbie here. I want to make a XCH transaction. How much should I pay for transaction fees for 2 xch for example? Can I leave it blank or if I want to pay 0.01, should have to use dot or comma? Like 0.01 or 0,01 thanks!

**Comment by tiny0little** [comment]
you can pay '0' or more, in a format with a dot and digits

**Comment by BrandtH22** [comment]
@lacrymatoire , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
